tsuru/main.go

```diff
@@ -59,7 +59,6 @@ func buildManager(name string) *cmd.Manager {
 	m.Register(&client.ServiceInstanceUpdate{})
 	m.Register(&client.ServiceInstanceRemove{})
 	m.Register(client.ServiceInfo{})
-	m.Register(client.ServiceInstanceInfo{})
 	m.Register(client.ServiceInstanceStatus{})
```

Comment:
Could you also remove `client.ServiceInstanceStatus{}` from here and delete its implementation from the services.go file?

Reply:
Removed the client.ServiceInstanceStatus implementation.